tests/e2e/specs/user-input-questions.test.js

```diff
@@ -36,66 +36,52 @@ import {
 	setSearchConsoleProperty,
 } from '../utils';
 
-const wpConditionalDescribe = () => {
-	// Only run for non WP 4.x versions
-	if ( process.env.WP_VERSION?.[ 0 ] !== '4' ) {
-		return describe;
-	}
-	return () => {
-		it( 'is skipped for WP 4.x', () => {} );
-	}; // no-op
-};
-
-wpConditionalDescribe()( 'User Input Settings', () => {
+describe( 'User Input Settings', () => {
 	async function fillInInputSettings() {
-		await step( 'select role', async () => {
-			await page.waitForSelector( '.googlesitekit-user-input__question' );
-			await expect( page ).toClick( '#role-owner_with_team' );
+		await page.waitForSelector(
+			'.googlesitekit-user-input__question--active'
+		);
+
+		await step( 'select purpose', async () => {
+			await expect( page ).toClick( '#purpose-publish_blog' );
 		} );
 
+		await expect( page ).toClick(
+			'.googlesitekit-user-input__question--active .googlesitekit-user-input__buttons--next'
+		);
+
+		await pageWait();
```

Comment:
These were added between steps which seemed to fix the problem although not exactly sure why. This is often the case with this utility. It could be that the choices are initially disabled for a very brief time and the added delay avoids that (which also somewhat explains why this would pass in an interactive run as each step gets 80ms delay in that mode).